docs/api/index.md

```diff
@@ -746,12 +746,18 @@ routes: RouteRecordRaw[]
 
 ### scrollBehavior
 
-Function to control scrolling when navigating between pages. Can return a Promise to delay scrolling. Check .
+Function to control scrolling when navigating between pages. Can return:
+  - a ScrollPosition object, which can contain:
+    - [ScrollToOptions](https://developer.mozilla.org/en-US/docs/Web/API/ScrollToOptions)
+    - an `el` property with a selector or reference to an element
+  - a Promise to delay scrolling
```

Comment:
Can you revert this change? This is already included in the guide

Reply:
Just added a small bit of extra detail and removed the incomplete sentence.